requirements.txt

```diff
@@ -16,7 +16,7 @@ python-multipart
 python-docx
 sqlalchemy_utils
 # for using FAISS with GPUs, install faiss-gpu
-faiss-cpu
+faiss-cpu; sys_platform != 'win32' and sys_platform != 'cygwin'
```

Comment:
Refer https://www.python.org/dev/peps/pep-0508/

Reply:
@lalitpagaria did you test if we need to add exception handling for the import statements in the `FAISSDocumentStore`? Otherwise, the installation would succeed but loading Haystack might fail.

Reply:
@tanaysoni I dont have windows system so did not tested on it, hence as requested @Zenahr and @sophgit to test changes on windows platform by running -
```
pip install git+https://github.com/lalitpagaria/haystack.git@fix_windows_installation
```

Current `haystack` rest API is not using `FAISSDocumentStore` so it might not fail there to load. But I agree we can add comment and raise exception with conditional import. WDYT?

Reply:
Added exception if user try to use or import FAISSDocumentStore